README.md

```diff
@@ -12,6 +12,7 @@ Initialize structs with default values
   - Scalar types
     - `int/8/16/32/64`, `uint/8/16/32/64`, `float32/64`
     - `uintptr`, `bool`, `string`
+      - `bool` values declared as a `*bool` allow for an unset boolean type. This helps distinguish the case where a default true bool is set to `false`.
```

Comment:
This is not exclusive to bool values.
Any scalar type can't distinguish an unspecified value from its zero value.
For int, an unspecified value means `0`; For string, an empty string (`""`).

If we want to indicate this behavior in README, we might as well do with technicalities.